samples/_bootstrap.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import re
+
+STORAGE_ACCOUNT_NAME = os.environ.get("AZSTORAGETORCH_STORAGE_ACCOUNT_NAME")
+CONTAINER_NAME = os.environ.get("AZSTORAGETORCH_CONTAINER_NAME")
+
+
+def modify_url(path):
+    if not STORAGE_ACCOUNT_NAME or not CONTAINER_NAME:
+        print(
+            "Please set environment variables AZSTORAGETORCH_STORAGE_ACCOUNT_NAME and AZSTORAGETORCH_CONTAINER_NAME"
+        )
+        return
+
+    with open(path, "r") as f:
+        content = f.read()
+
+    pattern = r'(CONTAINER_URL\s*=\s*\(\s*"https://)([^.]+)(\.blob\.core\.windows\.net/)([^"]+)("\s*\))'
+    new_url = rf"\1{STORAGE_ACCOUNT_NAME}\3{CONTAINER_NAME}\5"
+
+    return re.sub(pattern, new_url, content)
+
+
+def run_sample(path):
+    modified_content = modify_url(path)
+    if modified_content:
+        local_vars = {}
+        exec(modified_content, locals=local_vars)
+
+        for var_name, var_value in local_vars.items():
+            if var_name == "map_dataset" or var_name == "iterable_dataset":
+                for item in var_value:
+                    print(item)
```

Comment:
Then if we switch to using subprocesses, we should just make sure the subprocess exited with a return code of zero, which we should be able to turn on with `check=True`.